frontend/wallet/src/render/views/transactions/Transactions.re

```diff
@@ -172,6 +172,7 @@ let make = () => {
        let transactionQuery =
          TransactionsQueryString.make(
            ~publicKey=Apollo.Encoders.publicKey(pubkey),
+           ~after="",
```

Comment:
is this required?

Reply:
Yes, this fixes a bug in the Transactions view. Maybe related to upgrading bs-platform?